src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -664,6 +698,199 @@ void onBestBlock_whenPegoutTxIsCachedWithInvalidTimestamp_shouldSignSamePegoutTx
         );
     }
 
+    @Test
+    void onBestBlock_whenOnlySvpSpendTxWaitingForSignaturesIsAvailable_shouldAddSignature() throws Exception {
+        // Arrange
+        Federation federation = TestUtils.createFederation(params, 9);
+        FederationMember federationMember = federation.getMembers().get(0);
```

Comment:
i know is the same since we are using mocks and etc, but it would be more realistic if we use a _proposed_ fed member when signing the spend tx. Wdyt?